packages/sdk/src/__tests__/__snapshots__/handlePanic.test.ts.snap

```diff
@@ -1,11 +1,13 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`handlePanic ask to submit the panic error in interactive mode 1`] = `
+exports[`handlePanic when sendPanic fails, the user should be alerted by a reportFailedMessage 1`] = `
 "Oops, an unexpected error occured!
-Some error message!
```

Comment:
Not sure if this (outputting the original error I think) is functionality that got lost along the way?

Reply:
Nice catch, I will investigate this tomorrow

Reply:
Ok, I think I have figured this out: long story short, no functionality got lost, but we mocked a bit too aggressively.

In particular:
- in the `when sendPanic fails, the user should be alerted by a reportFailedMessage` test case, we mock `sendPanic` and we make it fail
-  we then call `expect(stripAnsi(ctx.mocked['console.log'].mock.calls.join('\n'))).toMatchSnapshot()`, which has created the snapshot file above
- since we mock `sendPanic`, this [`debug line here`](https://github.com/prisma/prisma/blob/fix/report-failed-message-is-never-displayed/packages/sdk/src/sendPanic.ts#L92) doesn't get executed

Reply:
As not clear from the message: Was this mocking changed to make sure we actually log the original error message? Or is that tested somewhere else?